lib/harness/ssh.js

```diff
@@ -54,11 +55,13 @@ class SSHConnector {
         return await this._JSSSHExec(cmd, this.sshConfig);
     }
 
-    async _JSSSHExec( cmd, sshConfig, timeout=20000, verbose=false, options={} ) 
-    {
+    async _JSSSHExec( cmd, sshConfig, timeout = 5000, verbose = false, options = {
+        count: 0
```

Comment:
I think it would be better to use the `options` object, `{count: X}`, as the number of times the caller wants to try. Right now it's being used just as a counter variable and number of tries is hard coded in the function (`options.count === 6`).